app/help/en/introduction/bitshares.md

```diff
@@ -1,56 +1,64 @@
 # BitShares
+
 This is the reference wallet of BitShares Blockchain which connects to BitShares Blockchain nodes (bitshares-core).
 
 This reference wallet is a browser based wallet. Your keys are stored locally in your browser and never leave it as transactions are signed locally 
 before broadcasting them. Never expose your keys to anyone. 
 
+
+## Introduction
+
+BitShares Blockchain is a decentralized “Blockchain As Organization” (BAO); an industrial-grade decentralized Web 3.0 platform built to run high-performance financial means and smart contracts. It represents the first decentralized autonomous community that enables its core token holders to decide its future direction and financial products.
```

Comment:
This is a term that I'd created alternative to *autonomous community* because I see it having more features than an *autonomous community* such as *worker system* to manage *Human Resources* and *Vesting*.